gms/gossiper.cc

```diff
@@ -2121,18 +2121,20 @@ void gossiper::build_seeds_list() {
     }
 }
 
-future<> gossiper::add_saved_endpoint(locator::host_id host_id, inet_address ep, permit_id pid) {
+future<> gossiper::add_saved_endpoint(locator::host_id host_id, gms::loaded_endpoint_state st, permit_id pid) {
```

Comment:
` gossiper: add_saved_endpoint: set dc and rack `

I don't see the logic which sets DC and Rack in this commit

Reply:
Odd.  It was probably got dropped by mistake when rebasing.
This PR had too many versions....
I'll add it back of course (and retest)

Reply:
restored in v11